src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/Miscellaneous/MarshalClassTests.cs

```diff
@@ -0,0 +1,188 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
Suggestion: Would you split this file into couple files: such as SizeOfTests.cs,StructureToPtrTests.cs

Reply:
Splited into PtrToStringTest.cs, PtrToStructureTests.cs, SizeOfTests.cs & DestroyStructureTests.cs.
And some tests are moved to existing tests like OffSetOfTests.cs & StructureToPtrTests.cs